dds/DCPS/DiscoveryBase.h

```diff
@@ -1612,6 +1641,31 @@ namespace OpenDDS {
       DDS::BuiltinTopicKey_t pub_bit_key_, sub_bit_key_;
       XTypes::TypeLookupService_rch type_lookup_service_;
 
+      struct GuidPrefixWrapper {
```

Comment:
I think there is no advantage. I wanted to use `GuidPrefix_t` as key so that it's more clear that each entry in the map is for a remote participant. But using GUID works too. Do you want to change it to use GUID as key?